### DISCUSSION TITLE
My server crash player if go into crash wall

### DISCUSSION THREAD

**Discussion by ghost** [Pryaxis/TShock#1709] [General]
Last night I was hacked into a server and spamed a lot of strange characters with numbers, and then he threw away a lot of stuff.Is there any way to prevent spam and fix the 2-wall error when going into causing game crashes
Not all crash the game, only those who go into that area.

* TShock version: New
* TShock build number (if known):
![Screenshot_2020-04-25-07-23-17-573_com discord](https://user-images.githubusercontent.com/46393687/80267007-270e0480-86c9-11ea-89af-d11d2a451ba4.jpg)
![Screenshot_2020-04-25-07-25-23-318_com and games505 TerrariaPaid](https://user-images.githubusercontent.com/46393687/80267009-2a08f500-86c9-11ea-9973-cd8a1ab713ca.jpg)
![Screenshot_20200425072132](https://user-images.githubusercontent.com/46393687/80267011-2e351280-86c9-11ea-88d3-fdd49d6fa035.jpg)

**Comment by moisterrific** [comment]
Mobile versions are not officially supported and it’s very likely mobile related issues have already been patched in the current PC release. 

The bi-directional Discord relay is more of an issue with the plugin you used to enable that functionality than TShock itself.

**Comment by QuiCM** [comment]
As mentioned by moist, we don't support mobile Terraria. Try editing the world file in a program like TEditor and removing the tiles in the area that is causing crashes

**Comment by bartico6** [comment]
As a side note, you should copy the world with the "crash wall" onto a computer and see if it causes crashes on Desktop 1.3.5.3 as well. If it does, create an issue so it can be fixed.
